client/src/docs-ui/page/page.tsx

```diff
@@ -101,7 +101,7 @@ export class DocsPage {
 
   @Listen("resize", {target: "window"})
   setSidebarStickyTop() {
-    if (this.pageData?.menu) {
+    if (Build.isBrowser && this.pageData?.menu) {
```

Comment:
When is the build not a browser?

Reply:
When prerendering and running tests -- I don't think there's a way to distinguish between the two.